src/plugins/condition/ConditionManager.js

```diff
@@ -31,13 +31,41 @@ export default class ConditionManager extends EventEmitter {
         this.conditionSetDomainObject = conditionSetDomainObject;
         this.timeAPI = this.openmct.time;
         this.latestTimestamp = {};
+        this.composition = this.openmct.composition.get(conditionSetDomainObject);
+        this.composition.on('add', this.addTelemetry, this);
+        this.composition.on('remove', this.removeTelemetry, this);
+        this.subscriptions = {};
         this.initialize();
 
         this.stopObservingForChanges = this.openmct.objects.observe(this.conditionSetDomainObject, '*', (newDomainObject) => {
             this.conditionSetDomainObject = newDomainObject;
         });
     }
 
+    addTelemetry(endpoint) {
```

Comment:
Probably should rename this from `addTelemetry` to `subscribeToTelemetry` and for `removeTelemetry` to `unsubscribeFromTelemetry`

Reply:
sure